riscv64/start_in_qemu.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -ex
+
+$QEMU_DIR/bin/qemu-system-riscv64 \
+    -M virt,aclint=on,aia=aplic-imsic -nographic \
+    -smp 3 -m 6G \
```

Comment:
I just checked what type of AWS instances our CI is using (c5d.metal) and how many buildkite agents we pack per metal (5), so here we can use 
```suggestion
    -smp 16 -m 32G \
```
and still have a bit of vcpu and memory to spare outside of the VMs :)

Reply:
Noted, waiting to see the results of introduces `vhost-user` related dependencies, I will update it in next force-push